### PR TITLE
use cross-env for webpack

### PR DESCRIPTION
**Pull request by mikekidder**
Worked for me in Windows!!  Give it a whirl..

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  },
+  resolve: {
+    extensions: ['', '.js']
+  },
+  plugins : [
+    new webpack.optimize.OccurenceOrderPlugin()
+  ]
+};
+
+if (process.env.NODE_ENV == 'production'){
```
Comment (gaearon) [inline webpack.config.js]: Nit: needs space before `{`

File: webpack.config.production.js [hunk 2]
```diff
@@ -1,20 +0,0 @@
-'use strict';
-
-var webpack = require('webpack');
-var baseConfig = require('./webpack.config.base');
-
-var config = Object.create(baseConfig);
-config.plugins = [
-  new webpack.optimize.OccurenceOrderPlugin(),
-  new webpack.DefinePlugin({
```
Comment (gaearon) [inline webpack.config.production.js]: Don't we still need `DefinePlugin`?

File: package.json [hunk 3]
```diff
@@ -18,8 +18,8 @@
     "test:examples": "babel-node examples/testAll.js",
     "check": "npm run lint && npm run test",
     "build:lib": "babel src --out-dir lib",
-    "build:umd": "webpack src/index.js dist/redux.js --config webpack.config.development.js",
-    "build:umd:min": "webpack src/index.js dist/redux.min.js --config webpack.config.production.js",
+    "build:umd": "cross-env NODE_ENV=dev webpack src/index.js dist/redux.js",
```
Comment (gaearon) [inline package.json]: Let's call it `development` rather than `dev`

File: webpack.config.production.js (hunk 2 above)
Comment (mikekidder) [inline webpack.config.production.js]: Don't believe so, since we are setting NODE_ENV now via npm script line

Comment (gaearon) [inline webpack.config.production.js]: We are setting NODE_ENV for Webpack itself (the Node part). But it won't be polyfilled in generated JS code (the client side) unless we tell webpack a la

```js
new webpack.DefinePlugin({
  'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV)
})
```

Comment (mikekidder) [inline webpack.config.production.js]: ah, yes needed then!  refreshed commit...

**Comment by mikekidder** [comment]
Let me know if you want me to merge the other cross-env PR #1313 
I'm playing around with git today.... rebase, squash, merge... fun, fun..

**Comment by gaearon** [comment]
Did the same in React Redux. Thank you for submitting this. Made my life easier!

**Comment by mikekidder** [comment]
Glad to help!

**Pull request merged**